adagrams/game.py

```diff
@@ -1,13 +1,107 @@
 from random import randint
 
 def draw_letters():
-    pass
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    pile = []
+    hand = []
+    # created a list of letters
+    for k, v in LETTER_POOL.items():  
+        for counter in range(0, v):      
+            pile.append(k)
+
+    for i in range(10):
+        index = randint(0, len(pile)-1)
```

Comment:
```suggestion
        index = randint(0, len(pile) -1 )
```
You're missing white spaces around the `-` operator. 
Take care to use white spaces correctly. We should surround operators like `-` with white spaces. You can review the PEP8 style guide on white spaces [here](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements). It's a small thing, but as you write more code (and eventually contribute to a large codebase) it's important to be consistent with whitespaces so that the whole project stays neat.

Reply:
While `index` does refer to a letter's index, more specifically it refers to a __random__ index. Prefer to name this variable a little more descriptively so other devs know what the variable references and it'll make your code more self-documenting. So a name like `random_letter_index` would be suitable.